app/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CannotConfirmBusinessErrorController.scala

```diff
@@ -64,13 +64,18 @@ class CannotConfirmBusinessErrorController @Inject()(mcc: MessagesControllerComp
                   formAction = routes.CannotConfirmBusinessErrorController.submit(journeyId),
                   form = formWithErrors
                 ))
-              , {
-                tryAgain =>
-                  if (tryAgain) Redirect(journeyConfig.continueUrl + s"?journeyId=$journeyId")
-                  else Redirect(routes.CaptureFullNameController.show(journeyId))
-              }
+              , tryAgain =>
+                if (tryAgain) {
+                  submissionService.submit(journeyId)
```

Comment:
We don't need this. It's already storing all the required fields when it calls submissionService.submit the first time round after CYA. Sorry - I didn't realise when I first suggested this